Controller/CategoryAdminController.php

```diff
@@ -99,7 +98,7 @@ public function treeAction(Request $request)
         $datagrid = $this->admin->getDatagrid();
 
         if ($this->admin->getPersistentParameter('context')) {
-            $datagrid->setValue('context', ChoiceType::TYPE_EQUAL, $this->admin->getPersistentParameter('context'));
+            $datagrid->setValue('context', null, $this->admin->getPersistentParameter('context'));
```

Comment:
Are you 100% sure? If yes, please pass `EqualType::TYPE_IS_EQUAL`, because it will help understand things faster if there ever is another bug in this area of the code.